rango-exchange/rango-client
# feat: add github action for sync translation with crowdin

# Summary

Adding a new workflow to GitHub Actions for syncing and keeping track of translated files, A new workflow(crowdin-action) has been added to the project that runs after each merge of a new branch into either the next or main branch. This workflow checks if the new text has been added to the project, send it to Crowdin, and then, in the next step, if there is a new translation in Crowdin, it downloads it. For the new changes, it creates a new pull request automatically and merges it automatically.

After the completion of this workflow(crowdin-action), the "publish" workflow is automatically executed and continues the previous process.

Additionally, there is the option to manually trigger the crowdin-action workflow if needed.


# How did you test this change?

- Fork from current project
- add CROWDIN_PROJECT_ID and CROWDIN_PERSONAL_TOKEN in secret settings.
- change some text in the project
- create new pr for changes
- merge PR to next 
- pull next on your branch
- clean and build project
- you should see translation in different languages


# Checklist:
